notebooks/EOPF_Tutorials/Sentinel-1_L2_OCN_product_format_prototype.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>Objectives and goals should be formatted in the same ways as in the other notebooks, see e.g. <a href="https://eopf-sample-service.github.io/eopf-sample-notebooks/sentinel-2-l1c-msi-zarr-product-exploration" target="_blank">https://eopf-sample-service.github.io/eopf-sample-notebooks/sentinel-2-l1c-msi-zarr-product-exploration</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>